lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Wallets.hs

```diff
@@ -1263,3 +1263,22 @@ spec = describe "SHELLEY_WALLETS" $ do
                     , expectField (#tip . #slotNumber  . #getApiT) (`shouldBe` slotNum)
                     , expectField (#tip . #height) (`shouldBe` blockHeight)
                     ]
+
+    it "SETTINGS_01 - Can put and read settings" $ \ctx -> do
```

Comment:
Maybe for better clarity it would be good to put test in a separate file? (For instance `Miscellaneous.hs`) or at least inside `StakePools.hs` to which it is more related with its current set of settings... :)